plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryTypeMapping.java

```diff
@@ -91,6 +101,126 @@ public void testNumericMapping()
                 .execute(getQueryRunner(), bigqueryCreateAndInsert("test.numeric"));
     }
 
+    @Test(dataProvider = "sessionZonesDataProvider")
+    public void testDatetime(ZoneId sessionZone)
```

Comment:
BQ tests can be expensive to run (inserts are expensive, right?)
and we believe session time zone should not affect results anyway. Let's maybe run them in single session

Reply:
Inserts are little expensive comparing to other RDBMS. Changed to run in a session.